signer/src/ecdsa.rs

```diff
@@ -192,31 +196,36 @@ pub fn verify<M: AsRef<[u8]>>(sig: &Signature, message: M, pubkey: &PublicKey) -
     };
     let message_hash = sp_core_hashing::blake2_256(message.as_ref());
     let wrapped = Message::from_digest_slice(&message_hash).expect("Message is 32 bytes; qed");
-    signature.verify(&wrapped, &public).is_ok()
+
+    Secp256k1::verification_only()
+        .verify_ecdsa(&wrapped, &signature, &public)
+        .is_ok()
 }
 
 /// An error handed back if creating a keypair fails.
-#[derive(Debug, PartialEq, thiserror::Error)]
+#[derive(Debug, PartialEq, Display, From)]
```

Comment:
impl StdError for std?

```rust
#[cfg(feature = std)]
impl std::error::Error  for Error {}
```

Reply:
What do you mean by this?

Reply:
I mean that the error type doesn't implement std::error::Error anymore.

So, do we want implement for std::error::Error if --feature std is enabled on the crate?

Reply:
Oh true! This is a really good catch.